db/seeds/create_content_blocks.rb

```diff
@@ -0,0 +1,258 @@
+# rubocop:disable Layout/HeredocIndentation
+
+markdown_for_other_useful_resources = <<-MARKDOWN_FOR_OTHER_USERFUL_RESOURCES
+## Other useful resources
+[Changes to the early years foundation stage framework](https://www.gov.uk/government/publications/changes-to-the-early-years-foundation-stage-eyfs-framework/changes-to-the-early-years-foundation-stage-eyfs-framework)
+
+[Statutory framework for the early years foundation stage](https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2)
+
+[Development Matters, non-statutory curriculum guidance for the early years foundation stage ](https://www.gov.uk/government/publications/development-matters--2)
+
+
+MARKDOWN_FOR_OTHER_USERFUL_RESOURCES
+
+unless ContentBlock.exists?(name: "other_useful_resources")
+  ContentBlock.create! do |u|
+    u.name = "other_useful_resources"
+    u.description = "Used on the landing page to describe Other Useful Resources"
+    u.markdown = markdown_for_other_useful_resources
+  end
+end
+
+html_for_accessibility = <<-HTML_FOR_ACCESSIBILITY
+<h1 class="govuk-heading-xl">Accessibility statement for the Help for early years providers service</h1>
+
+This accessibility statement applies to the Help for early years providers service.
+
+This website is run by the Department for Education (DfE). We want as many people as possible to be able to use this website. For example, that means you will be able to:
+
+* change colours, contrast levels and fonts
+* zoom in up to 300% without the text spilling off the screen
+* navigate the website using just a keyboard
+* navigate the website using speech recognition software
+* listen to the website using a screen reader (including the most recent versions of JAWS and NVDA)
+
+We’ve also made the website text as simple as possible to understand.
+
+# How accessible this website is
+
```

Comment:
This should be a h2 

Darren noticed,  'How accessible this website is' within seed is down as h1 but should be h2.

Reply:
ok, updated